src/routes/companies/company.industry.ts

```diff
@@ -0,0 +1,110 @@
+import express from 'express'
+import { z } from 'zod'
+import { processRequest } from '../zod-middleware'
+import { createIndustry, updateIndustry } from '../../lib/prisma'
+import { wikidataIdParamSchema } from '../../openapi/schemas'
+import { prisma } from '../../lib/prisma'
+import { GarboAPIError } from '../../lib/garbo-api-error'
+
+const router = express.Router()
+
+const industrySchema = z.object({
+  industry: z.object({
+    subIndustryCode: z.string(),
+  }),
+})
+
+/**
+ * @swagger
+ * /companies/{wikidataId}/industry:
+ *   post:
+ *     summary: Update company industry classification
+ *     description: Update or create industry classification for a company
+ *     tags: [Companies]
+ *     parameters:
+ *       - in: path
+ *         name: wikidataId
+ *         required: true
+ *         schema:
+ *           type: string
+ *         description: Wikidata ID of the company
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             required:
+ *               - industry
+ *             properties:
+ *               industry:
+ *                 $ref: '#/components/schemas/IndustrySchema'
+ *     responses:
+ *       200:
+ *         description: Industry updated successfully
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 ok:
+ *                   type: boolean
+ *       404:
+ *         description: Company not found
+ *       422:
+ *         description: Validation error
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/Error'
+ *       500:
+ *         description: Server error
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/Error'
+ */
+router.post(
```

Comment:
Future idea:

As we discussed, we might want to generate the OpenAPI docs in the background so we don't need to keep it as code comments.

We could also setup response validation to make sure we return data in the expected format, thus making the the API docs more reliable and easier to work with. I have a solution in the backend of https://github.com/paccao/allerthsbageri.se/tree/feat/booking-system that we could use.